source/common/grpc/common.cc

```diff
@@ -57,10 +57,9 @@ void Common::chargeStat(const Upstream::ClusterInfo& cluster, const std::string&
                            grpc_status->value().getStringView()))
       .inc();
   uint64_t grpc_status_code;
-  const std::string grpc_status_string(grpc_status->value().getStringView());
   // TODO(dnoe): Migrate to pure string_view (#6580)
```

Comment:
@dnoe I think this can be removed now? Can you merge into one of your follow ups?

Reply:
Will do.